docs/core_docs/docs/modules/index.mdx

```diff
@@ -29,3 +29,7 @@ Persist application state between runs of a chain
 #### [Callbacks](/docs/modules/callbacks/)
 
 Log and stream intermediate steps of any chain
+
+#### [Experimental](/docs/modules/experimental/)
+
+Experimental modules not yet ready for production consumption
```

Comment:
I would phrase this more as `Experimental modules whose abstractions have not fully settled`